docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -13,11 +13,17 @@ The following table lists the available UI components:
 
 | Control                              | Description                              | Related classes                          |
 | ------------------------------------ | ---------------------------------------- | ---------------------------------------- |
-| [Buttons](./button.md)               | The Button class is a base class for different kinds of buttons. | `Button`, `PushButton`, `CheckBoxButton`, `RadioButton`, `ToggleButton` |
+| [Button](./nui-components/Button.md) | A button that can set action when user select it. | `Button`                |
```

Comment:
**Question** to this table list and items listed up in toc_all.md (under #### Components)

Is there any rule to list up these UI components?
If you don't have, how about following alphabetical rule?

**Suggestion** to the replaced UI components.
Do you want to keep old version of replaced UI components?
If not, how about deleting the files and the images used for previous Button, Slider, and Tab?


@rishu-samsung if you have other query, please let us know.

Reply:
Hello Ms. @mijin85cho, these updates seems fine. Also sorting these items alphabetically is much better to read and find, from an end user perspective. Please do the needful.

Reply:
I agree following alphabetical rule. We don't want to keep old version.